src/fury.js

```diff
@@ -43,7 +43,7 @@ class Fury {
    * then uses the adapter to convert into refract elements and loads
    * these into objects.
    */
-  parse({source, mediaType, generateSourceMap = false}, done) {
+  parse({source, mediaType, generateSourceMap = false, parserOptions}, done) {
```

Comment:
I do not really like the idea of yet another parameter as `generateSourceMap` is part of `parserOptions`, but probably it's the only way unless we want to have a breaking change.